core/src/main/kotlin/org/evomaster/core/search/gene/collection/ArrayGene.kt

```diff
@@ -257,6 +268,51 @@ class ArrayGene<T>(
                 closingTag
     }
 
+    override fun getDtoCall(dtoName: String, counter: Int): DtoCall {
```

Comment:
here we are dealing with code generation. you have declaration of varibles and statement ending with `;`. it should not be inside `Gene`. these functionalities should be refactored out somewhere under the `output` package